security/external_secrets_operator/external-secrets-operator-proxy.adoc

```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="external-secrets-operator-proxy"]
+= Egress proxy for the External Secrets Operator for Red Hat OpenShift
+include::_attributes/common-attributes.adoc[]
+:context: external-secrets-operator-proxy
+
+If a cluster-wide egress proxy is configured in {product-title}, Operator Lifecycle Manager (OLM) automatically configures Operators that it manages with the cluster-wide proxy. OLM automatically updates all of the Operator's deployments with the `HTTP_PROXY`, `HTTPS_PROXY`, `NO_PROXY` environment variables.
+
+Also, the proxy can be configured in the `ExternalSecretsConfig` or `ExternalSecretsManager` custom resources.
+
+The proxy configuration is determined by the following order of precedence, from highest to lowest:
+
+. `externalsecretsconfigs.operator.openshift.io`
+
+. `externalsecretsmanager.operator.openshift.io`
+
+. Proxy set by OLM
+
+// Configuring external secrets operator proxy
+include::modules/external-secrets-proxy-support.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="external-resources-operator-proxy_additional-resources"]
+== Additional resources
+
+* xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[Configuring proxy support in Operator Lifecycle Manager]
+
+
+// Proxy security considerations
+include::modules/external-secrets-proxy-security-considerations.adoc[leveloffset=+2]
```

Comment:
We need to add it as a sub-section under `11.1. External Secrets Operator for Red Hat OpenShift overview`